zsh-bash-completions-fallback.plugin.zsh

```diff
@@ -91,7 +97,17 @@ function _bash_completions_fetch_supported_commands {
 }
 
 function _bash_completions_load {
-    local bash_completions=${ZSH_BASH_COMPLETIONS_FALLBACK_PATH:-${${(@s/:/)${XDG_DATA_DIRS:-/usr/share}}[1]}/bash-completion}
+    local dir bash_completions
+    if [ -d "$ZSH_BASH_COMPLETIONS_FALLBACK_PATH" ]; then
+        bash_completions=$ZSH_BASH_COMPLETIONS_FALLBACK_PATH
+    else
+        for dir in "${_bash_completions_dirs[@]}"; do
+            if [ -d $dir/bash-completion ]; then
+                bash_completions=$dir/bash-completion
+                break
+            fi
+        done
+    fi
```

Comment:
Mhmh, I feel we shoulld move the old content of `_bash_completions_load` to `_bash_completions_load_from_dir` and the en actually call it for each `$dir/bash-completion`, no?

Reply:
What is the difference? I think both of them can work.